### PR TITLE
20 backend json file

### PR DESCRIPTION
**Pull request by ugedal1**
## Description
- Locations now gets loaded from a json file with location names and roles. The roles gets chosen at random and given out.
- Location model created with name and list of roles
- 
- Split Lobbyservice into lobbyManagerService and lobbyHostService, this is meant to give further seperation and make it a little cleaner.

- Fixed almost all tests, but 4 doesnt pass, tried to fix but used to much time, but tested manually so should work.

### Type of change

- [ ] Bug fix 
- [ ] New functionality 
- [ ] Design change
- [ ] New tests
- [ ] Frontend change
- [ ] Backend change

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] The solution is responsive (remove if not relevant)
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added tests for edge cases
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: interloperServer/src/main/java/com/interloperServer/interloperServer/service/GameService.java [hunk 1]
```diff
@@ -93,7 +94,7 @@ public void handlePlayerDisconnect(WebSocketSession session, String lobbyCode) {
             return;
         }
 
-        lobbyService.removeUser(session);
+        lobbyManager.removeUser(session);
         // If there is less than 2 left, end it
         if (game.getPlayers().size() < 2) {
```
Comment (supergelf) [inline interloperServer/src/main/java/com/interloperServer/interloperServer/service/GameService.java:99]: Kanskje også endre dette til 3, siden man må være 3 for å starte spillet

**Review by supergelf — APPROVED** [review]

File: interloperServer/src/main/java/com/interloperServer/interloperServer/service/GameService.java (hunk 1 above)
Comment (ugedal1) [reply interloperServer/src/main/java/com/interloperServer/interloperServer/service/GameService.java:99]: gir mening det!
